api/graphql/makeModels.js

```diff
@@ -244,20 +247,23 @@ export default async function makeModels (userId, isAdmin) {
       ],
       getters: {
         feedItems: (c, args) => c.feedItems(args),
+        isPublic: c => c.get('is_public'),
```

Comment:
why do you need this, shouldn't all regular columns automatically be available to access?